examples/src/components/PivotTableDrillExample.jsx

```diff
@@ -34,9 +34,20 @@ export class PivotTableDrillExample extends Component {
         return true;
     }
 
-    render() {
+    renderDrillValue = () => {
```

Comment:
why arrow function ? :) I would rather to keep save function definition :) if it's not necessary